src/tests/ftest/dbench/dbench.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+from ClusterShell.NodeSet import NodeSet
+from dfuse_test_base import DfuseTestBase
+from command_utils import CommandFailure
+from dbench_utils import Dbench
+
+class DbenchTest(DfuseTestBase):
+    # pylint: disable=too-few-public-methods,too-many-ancestors
+    """Base Dbench test class.
+
+    :avocado: recursive
+    """
+
+    def test_dbench(self):
+        """Jira ID: DAOS-4780
+
+        Test Description:
+            Purpose of this test is to mount dfuse and run
+            dbench on top of it.
+
+        Use cases:
+            Create Pool.
+            Create container.
+            Mount dfuse.
+            Run dbench on top of mount point.
+
+        :avocado: tags=all,hw,daosio,medium,ib2,full_regression,dbench
```

Comment:
This doesn't seem to fit the newly decided/discussed multi-line tag format.

Reply:
Done